duckduckgo/iOS
# Sync management UI

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203756800930489/f
Tech Design URL:
CC:

**Description**:

Adds management screens for Sync

**Steps to test this PR**:
1. Smoke test entire UI
2. Ensure management screens show and don't have any obvious errors
3. Check for broad alignment with the designs (see task for link)



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
